src/GSM.cpp

```diff
@@ -123,6 +125,25 @@ int GSM::ready()
   }
 
   switch (_readyState) {
+    case READY_STATE_DISABLE_MESSAGE_WAITING_INDICATION: {
+      MODEM.send("AT+UMWI=0");
```

Comment:
Should we add a comment clarifying why we are disabling this :slightly_smiling_face:  ? Could someone may want it back for some reason?

Reply:
I don't think so, the only thing I can think of is a voice mail indication bot that blink's a LED to indicate you can a voice mail, however you still need to do something to retrieve it.

I think we can drop this commit, because the issues I was seeing last week were most likely related to using a SAMD core without flow control enabled.

Reply:
I've removed this change.